GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs

```diff
@@ -352,4 +360,4 @@ public static void RestorePurchase()
                 }
             });
     }
-}
+}
```

Comment:
nit: always a newline at the end of a file

Reply:
I use the Rider Reformat code, and it removes the newline for me.